### PR TITLE
Release notes for edge-18.11.3

### PR DESCRIPTION
**Pull request by siggy**
Signed-off-by: Andrew Seigner <siggy@buoyant.io>

Assumes #1881 will merge prior to release.

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,21 @@
+## edge-18.11.3
+
+* CLI
+  * **New** `linkerd routes` command displays per-route stats for services with
+    service profiles
+  * **Improved** Add `--open-api` to `linkerd profile` command, to read an
```
Comment (klingerf) [inline CHANGES.md]: Hmm, maybe: Add `--open-api` _flag_ to `linkerd profile` command, ...

File: CHANGES.md [hunk 2]
```diff
@@ -1,3 +1,21 @@
...
+    OpenAPI / Swagger specification file
+  * **Experimental** New `--ha` flag on `linkerd install`, for HA deployment of
```
Comment (klingerf) [inline CHANGES.md]: The rest of our bullet points start with a verb, so for consistency I'd change this to: Add `--ha` flag to `linkerd install` command, ...

File: CHANGES.md [hunk 3]
```diff
@@ -1,3 +1,21 @@
...
+    the control plane (thanks @benjdlambert!)
+* Web UI
+  * **New** Top Routes page, served at `/routes`
```
Comment (rmars) [inline CHANGES.md]: TIOLI might want to say "experimental" top routes page, as it's unpolished? Or we can just not mention it.

Comment (wmorgan) [inline CHANGES.md]: Any contributors we want to call out for this?

Comment (siggy) [reply CHANGES.md]: The `--ha` flag was from a community member, they get a call out.

**Pull request merged**